pkgs/servers/mail/mailman/web.nix

```diff
@@ -3,26 +3,33 @@
 }:
 
 buildPythonPackage rec {
-  pname = "mailman-web-unstable";
-  version = "2019-09-29";
+  pname = "mailman-web";
+  version = "unstable-2021-04-10";
   disabled = !isPy3k;
 
   src = fetchgit {
     url = "https://gitlab.com/mailman/mailman-web";
```

Comment:
This could use fetchFromGitLab.

Reply:
Unfortunately not, because `fetchFromGitLab` doesn't support `leaveDotGit`.